js/apps/admin-ui/src/groups/GroupsModal.tsx

```diff
@@ -132,15 +134,17 @@ export const GroupsModal = ({
         });
       }
 
-      const permissions = await adminClient.groups.listPermissions({
-        id: sourceGroup.id!,
-      });
+      if (isFeatureEnabled(Feature.AdminFineGrainedAuthzV2)) {
```

Comment:
I believe it should be guarded against V1:
```suggestion
      if (isFeatureEnabled(Feature.AdminFineGrainedAuthz)) {
```